backend/src/apiserver/resource/model_converter.go

```diff
@@ -167,7 +167,7 @@ func (r *ResourceManager) ToModelJob(job *api.Job, swf *util.ScheduledWorkflow,
 	} else if templateType == template.V2 {
 		params, err := runtimeConfigToModelParameters(job.GetPipelineSpec().GetRuntimeConfig())
 		if err != nil {
-			return nil, util.Wrap(err, "Unable to parse the parameter.")
+			return nil, util.Wrap(err, "Unable to parse the V2 parameter.")
```

Comment:
I would avoid using v1 vs v2 in error message. It would be confusing to users. If you want to differentiate the two for debugging purpose, use something like "unable to parse runtime config parameters"

Reply:
Updated.